avr/bootloaders/HoodLoader2/makefile

```diff
@@ -11,7 +11,9 @@
 
 # Run "make help" for target help.
 
+ifeq ($(MCU),)
```

Comment:
`?=` looks fancier. Please use that, its also more simple and better understandable.